specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/examples/sapvirtualinstances/SAPVirtualInstances_Create_Distributed_WithRestrictedDBVmSku.json

```diff
@@ -0,0 +1,339 @@
+                  "osConfiguration": {
+                    "osType": "Linux",
+                    "disablePasswordAuthentication": true,
+                    "ssh": {
+                      "publicKeys": [
+                        {
+                          "keyData": "ssh-rsa public key"
+                        }
+                      ]
+                    }
+                  }
+                }
+              },
+              "instanceCount": 1
+            },
+            "applicationServer": {
+              "subnetId": "/subscriptions/49d64d54-e966-4c46-a868-1999802b762c/resourceGroups/dindurkhya-e2etesting/providers/Microsoft.Networks/virtualNetworks/test-vnet/subnets/appsubnet",
+              "virtualMachineConfiguration": {
+                "vmSize": "Standard_E32ds_v4",
+                "imageReference": {
+                  "sku": "7.4",
+                  "publisher": "RedHat",
+                  "version": "7.4.2019062505",
+                  "offer": "RHEL-SAP"
+                },
+                "osProfile": {
+                  "adminUsername": "{your-username}",
+                  "osConfiguration": {
+                    "osType": "Linux",
+                    "disablePasswordAuthentication": true,
+                    "ssh": {
+                      "publicKeys": [
+                        {
+                          "keyData": "ssh-rsa public key"
+                        }
+                      ]
+                    }
+                  }
+                }
+              },
+              "instanceCount": 6
+            },
+            "databaseServer": {
+              "databaseType": "HANA",
+              "subnetId": "/subscriptions/49d64d54-e966-4c46-a868-1999802b762c/resourceGroups/dindurkhya-e2etesting/providers/Microsoft.Networks/virtualNetworks/test-vnet/subnets/dbsubnet",
+              "virtualMachineConfiguration": {
+                "vmSize": "Standard_E16ds_v4",
+                "imageReference": {
+                  "sku": "7.4",
+                  "publisher": "RedHat",
+                  "version": "7.4.2019062505",
+                  "offer": "RHEL-SAP"
+                },
+                "osProfile": {
+                  "adminUsername": "{your-username}",
+                  "osConfiguration": {
+                    "osType": "Linux",
+                    "disablePasswordAuthentication": true,
+                    "ssh": {
+                      "publicKeys": [
+                        {
+                          "keyData": "ssh-rsa public key"
+                        }
+                      ]
+                    }
+                  }
+                }
+              },
+              "instanceCount": 1
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/6d875e77-e412-4d7d-9af4-8895278b4443/resourceGroups/test-rg/providers/Microsoft.Workloads/sapVirtualInstances/X00",
```

Comment:
mismatching subscription, please make sure all details in request and response are same

Reply:
updated